internal/business/persisted_operations/gcp_storage_loader.go

```diff
@@ -33,8 +34,8 @@ func NewGcpStorageLoader(ctx context.Context, bucket string, store string) (*Gcp
 		store:  store,
 	}, nil
 }
-func (g *GcpStorageLoader) Load(ctx context.Context) error {
-	it := g.client.Bucket(g.bucket).Objects(ctx, &storage.Query{
+func (gcp *GcpStorageLoader) Load(ctx context.Context, log *slog.Logger) error {
```

Comment:
Go convention is to reference the struct on these methods using a single letter, like `g`

Reply:
i see, let me revert that